modules/accredited_representative_portal/db/migrate/20241212000300_add_indexes_to_ar_power_of_attorney_forms.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexesToArPowerOfAttorneyForms < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+  def change
+    add_index :ar_power_of_attorney_forms, :city_bidx, algorithm: :concurrently
+    add_index :ar_power_of_attorney_forms, :state_bidx, algorithm: :concurrently
+    add_index :ar_power_of_attorney_forms, :zipcode_bidx, algorithm: :concurrently
```

Comment:
I think we probably don't need it, but can you spend a little time convincing yourself whether or not we especially need [multicolumn indexes](https://www.postgresql.org/docs/current/indexes-multicolumn.html) to support our queries? Even if we get this wrong, it's no big deal, but good just to have it somewhere in the back of your mind.

Reply:
ok i changed it to multi-column and a single one for zip